esp8266/sonoffsc/mqtt.ino

```diff
@@ -176,3 +176,20 @@ void mqttLoop() {
     }
 
 }
+
+// Callback to subscrive 
+void mqttCallback(unsigned int type, const char * topic, const char * payload) {
+
+	// When connected, subscribe to the topic
+	if (type == MQTT_CONNECT_EVENT) {
+		mqttSubscribeRaw(MQTT_RGB_TOPIC);
```

Comment:
mqttSubscribe adds the root topic to the topic argument, whilst mqttSubscribeRaw doesn't. In general, if the topic is going to be specific for this device (in a home with several sonoffsc devices) it's better to user mqttSubscribe.